docs/integrations/destinations/s3-migrations.md

```diff
@@ -0,0 +1,203 @@
+| `{ "type": "array", "items": { "oneOf": [ {"type": "integer", "type": "string"} ] } }` | `[1, false]`   | `{ "type": "array", "items": [ "null", "integer", "string" ] }` | `[1, null*]`       |
+
+*The nulled fields represent conversion failures and will appear in `_airbyte_meta.changes[]`.
+
+This behavior will be applied to both Avro and Parquet formats.
+
+### Union Types (Parquet Only)
+
+#### Disjoint record improvements
+
+Formerly, unions in parquet were represented as anonymous disjoint records. For example:
+
+| Airbyte Union Type                                                                                                                     | Old Parquet Type                                                                                                         |
+|----------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| `{"oneOf": [ {"type": "integer"}, {"type": "string"} ] }`                                                                              | `Optional Record { member0: Optional Int32, member1: Optional String }`                                                  |
+| `{"oneOf": [ {"type": "boolean"}, {"type": "object", "properties": { "id": { "type": "integer" }, "name": { "type": string" } } } ] }` | `Optional Record { member0: Optional Boolean, member1: Optional Record { id: Optional Uint32, name: Optional String } }` |
+
+Now unions will be represented as typed disjoint records with named fields. For example:
+
+| Airbyte Union Type                                                                                                                     | New Parquet Type                                                                                                                               |
+|----------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| `{"oneOf": [ {"type": "integer"}, {"type": "string"} ] }`                                                                              | `Optional Record { type: Optional String, integer: Optional Int32, string: Optional String }`                                                  |
+| `{"oneOf": [ {"type": "boolean"}, {"type": "object", "properties": { "id": { "type": "integer" }, "name": { "type": string" } } } ] }` | `Optional Record { type: Optional String, boolean: Optional Boolean, object: Optional Record { id: Optional Uint32, name: Optional String } }` |
+
+Where
+* only one field will always be set, and
+* the value of the `type` field will always be equal the name of the set field
```

Comment:
```suggestion
* The value of the `type` field will always be equal the name of the `set `field.
```

Reply:
Not sure about putting ` around "set" -- there is not a field called set. `"the field that was set"` instead?